apps/github.js

```diff
@@ -27,14 +27,16 @@ async function mergePr(context, pr, onMergeFailure) {
   let i = 0
   let isMerged = false
   while (i++ < maxRetries) {
+    if (i !== 1) {
+      console.log(` Retrying... sleeping for ${(i + 2)} minute(s)`)
+      await timeout(60 * (i + 1) * 1000);
```

Comment:
@cb-shivamagarwal Octokit library will actually throw an error if it is unable a merge a PR. So, we should be wait inside the `catch`.

Reply:
I added it here because of the following thinking. In the `catch` block, we are logging the error. After catch block, it will run the loop again. At that time, it will wait before trying to merge again. 

However, if you still wish to wait in `catch` block, we can move the `if` block with a different condition inside the `catch` block.

Reply:
Yeah @cb-shivamagarwal semantically both are same. But the wait inside catch communicate the intent bit clearly from the readability perspective.

Reply:
Okay

Reply:
Done